hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/julinserg/go_home_work/tree/master/hw01_hello_otus
```

Рекомендую указать корректный путь до модуля, чтобы он был доступен для скачивания

```suggestion
module github.com/julinserg/go_home_work/hw01_hello_otus
```

---

Пофиксил, спасибо.